### PR TITLE
Update header text and call to action link

### PR DESCRIPTION
**Pull request by dimitraz**
**Description**
- Update header overview paragraph
- Update call to action link

### CODE REVIEW

File: _includes/homepage-header.html [hunk 1]
```diff
@@ -1,21 +1,15 @@
 <div class="row">
-
   <div class="col-sm-12">
-    <h1>Bringing <strong>mobile</strong> and <strong>enterprise</strong> together.</h1>
+    <h1>Cloud Native Mobile App Development on OpenShift.</h1>
```
Comment (JameelB) [inline _includes/homepage-header.html]: Can we remove the `.` here

File: _includes/homepage-header.html [hunk 2]
```diff
@@ -1,21 +1,15 @@
...
   </div>  
+      
   <div class="col-sm-10 col-sm-offset-1">
-    <p><strong>We make cross-platform enterprise mobile development easy</strong>. Aerogear provides flexible, extensible libraries and server side components that will simplify your mobile development and infrastructure setup across platforms.</p>
+    <p><strong>We make cross-platform enterprise mobile development easy</strong>. AeroGear provides a suite of Mobile Services across a range of native and hybrid client SDKs that will simplify your mobile and modern application development needs, leveraging OpenShift's Container technology to provide a secure, scalable backend platform.</p>
```
Comment (JameelB) [inline _includes/homepage-header.html]: Not sure about having the `<strong>` element here. To me, if this is meant to be highlighted, it kind of just disappears into the paragraph. Maybe we can add it as a sub-title instead? Kind of something like this? wdyt?
image

File: _includes/homepage-header.html (hunk 1 above)
Comment (dimitraz) [reply _includes/homepage-header.html]: :thinking:

File: _includes/homepage-header.html (hunk 2 above)
Comment (dimitraz) [reply _includes/homepage-header.html]: Yea agreed, it adds more emphasis

File: _includes/homepage-header.html (hunk 1 above)
Comment (jasonmadigan) [reply _includes/homepage-header.html]: +1 on ditching the `.`

**Comment by dimitraz** [comment]
@jasonmadigan @JameelB Updated, can you take another look?
image

**Comment by jasonmadigan** [comment]
@dimitraz perfect

**Pull request merged**